readme.md

```diff
@@ -75,7 +95,7 @@ Once you have successfully authenticated, make your user an admin: Start the rai
 
 ### Importing base data
 
-For proper functionality, you need to populate your database with *districts*.
+For proper functionality, you need to populate your database with _districts_.
```

Comment:
Sorry about these changes. Prettier taking over. Will add a CI check for prettier so this does not happen any more.

Reply:
Yeah, I am not 100 💯 sure about the state of js linting at the moment. I think rubocop and prettier did not like each other

Reply:
Also please only apply rules that are automatically fixable and do not contradict the ruby style

Reply:
It is possible to make all linters like each other. You just have to tell me what the issue is. There is a tool "ruby style" which now produces errors? How do I run this tool? I will add it to CI and make sure it is happy while keeping prettier and eslint happy as well.

Reply:
Lets continue this discussion in #699

Reply:
rubocop, see `.rubocop.yml` i'm a bit surprised its not run from ci 🤔

Reply:
Please lets continue this discussion in #699. Rubocop is already implemented there.